lib/chef/knife/core/bootstrap_context.rb

```diff
@@ -40,7 +40,16 @@ def initialize(config, run_list, chef_config, secret = nil)
         end
 
         def bootstrap_environment
-          @chef_config[:environment] || '_default'
+          # Check for environment in CLI, first_boot_attributes, then in the
+          # config file. Use '_default' if no environment specified.
+
+          # TODO: @chef_config picks up @config's environment setting, even when
```

Comment:
Is there anything to do for this `TODO` or is this just a future note of caution?